internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -105,7 +105,7 @@ func (e *EnvStackConfig) Template() (string, error) {
 		PrivateImportedCertARNs:  e.importPrivateCertARNs(),
 		VPCConfig:                e.vpcConfig(),
 		CustomInternalALBSubnets: e.internalALBSubnets(),
-		AllowVPCIngress:          e.in.AllowVPCIngress, // TODO(jwh): fetch AllowVPCIngress from Manifest or SSM.
```

Comment:
Do we need to remove `AllowVPCIngress` from `e.in`?

Reply:
+1

Reply:
we can't because we need to feed from `e.in` to manifest for `env show --manifest`